src/main/steps/applicant1/review-your-joint-application/template.njk

```diff
@@ -57,11 +57,7 @@
 
   <h2 class="govuk-heading-m">{{ heading6 }}</h2>
   <p class="govuk-body">{{ line6 }}</p>
-  <ul class="govuk-list govuk-list--bullet">
-    {% for connectionBullet in connectionBulletPoints %}
-      <li>{{ connectionBullet }}</li>
-    {% endfor %}
-  </ul>
```

Comment:
is this not displaying the correct text?

Reply:
it's already displayed as a list, so it doesn't need to be turned into one. If you look at the ticket you can see a screenshot of what it looked like before.

Reply:
is that because we're making it a list in the code rather than the template?

Reply:
yeah it was working on another page, but wasn't updated here